Rakefile

```diff
@@ -0,0 +1,62 @@
+version_string = `git describe --tag | cut  -d "-" -f 1,2 | tr - .`.chomp
+if version_string.empty?
+  version_string = '0'
+end
+date_string = Time.now.strftime("%Y-%m-%d")
+params = "--attribute revnumber='#{version_string}' --attribute revdate='#{date_string}'"
+
+image_files = Rake::FileList.new("src/images/*.png", "src/images/*.svg") do |fl|
+  fl.exclude("~*")
+  fl.exclude(/^scratch\//)
+end
+
+namespace :spec do
+  directory 'build/images'
+
+  desc 'copy images to build dir'
+  task :images => 'build/images'
+
+  image_files.each do |source|
+    target = source.sub(/^src\/images/, 'build/images')
+    file target => source do
+      cp source, target, :verbose => true
+      `pngquant -f #{target}`
+    end
+    desc "copies all data files"
+    task :images => target
+  end
+
+  task :prereqs => [:images]
+  
+  desc 'build basic spec formats'
+  task :html => :prereqs do
+    begin
+      puts "Converting to HTML..."
+      `bundle exec asciidoctor -b html5 #{params} src/main.adoc -o build/PartiQL-Specification.html`
+    end
+  end
+
+  task :pdf => :prereqs do
+    begin
+      theming = "-a pdf-themesdir=src/themes -a pdf-theme=basic -a pdf-fontsdir=build/fonts"
```

Comment:
Was getting the following error when building:

```
Converting to PDF...
bundler: failed to load command: asciidoctor-pdf (/Users/caialan/.rbenv/versions/3.1.2/bin/asciidoctor-pdf)
/Users/caialan/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/asciidoctor-pdf-2.3.0/lib/asciidoctor/pdf/converter.rb:4031:in `block (3 levels) in register_fonts': No such file or directory - vollkorn/TTF/Vollkorn-Regular.ttf not found in build/fonts (Errno::ENOENT)
```

Think this should point to `fonts` and not `build/fonts`. After I changed it to `fonts`, the PDF was properly generated but got this warning:

```
Converting to PDF...
asciidoctor: WARNING: Could not locate the character `' (\u0000) in the following fonts: Vollkorn, M+ 1mn
asciidoctor: WARNING: Could not locate the character `' (\u2063) in the following fonts: Vollkorn, M+ 1mn
```

Reply:
Alternatively, could change the install instructions to install the fonts into `partiql-spec/build/fonts/` rather than `partiql-spec/fonts/`.